UKHomeOffice/sgmr-service
# Fix radio button selection for charterOrHire

### AC / Description of changes
You couldn't select the `neither` option


### To Test
- create a new pleasurecraft
- go to page 2
- click on `neither` for charterOrHire
- > you should be able to select it
- complete form
- submit
- > it should submit without error
- return to that pleasure craft form (edit)
- > neither should be preselected
- change it to Hired
- submit
- return to form
- > hired should be preselected
- change to neither
- submit
- > it should submit without error
- return to form
- > neither should be preselected


### Notes

---

**Remember to merge to feature branches not `master`.**
